src/quo2/components/community/divider_label.cljs

```diff
@@ -0,0 +1,62 @@
+(ns quo2.components.community.divider-label
+  (:require [quo.react-native :as rn]
+            [quo2.components.markdown.text :as text]
```

Comment:
this might be confused with rn text, better to name it `markdown.text`

Reply:
done